koku/masu/util/gcp/common.py

```diff
@@ -4,27 +4,54 @@
 #
 """GCP utility functions and vars."""
 import datetime
-import json
 import logging
 import uuid
-from json.decoder import JSONDecodeError
 
-import ciso8601
 import pandas as pd
 from tenant_schemas.utils import schema_context
 
 from api.models import Provider
 from masu.database.provider_db_accessor import ProviderDBAccessor
 from masu.external.accounts_accessor import AccountsAccessor
 from masu.processor import disable_gcp_resource_matching
-from masu.util.common import safe_float
-from masu.util.common import strip_characters_from_column_name
 from masu.util.ocp.common import match_openshift_labels
 from reporting.provider.gcp.models import GCPCostEntryBill
 
 LOG = logging.getLogger(__name__)
 pd.options.mode.chained_assignment = None
 
+INGRESS_REQUIRED_COLUMNS = {
```

Comment:
I like the separation of values. I'm thinking there may be a cleaner way to do this than using the module namespace and a global, though. Still working on a coherent suggestion.